src/utils/formatDate.ts

```diff
@@ -5,5 +5,13 @@ import { format } from 'date-fns';
 
 export function formatDate(inputDate: string): string {
   const date = new Date(inputDate);
-  return format(date, 'yyyy-MM-dd');
+
+  try {
+    return format(date, 'yyyy-MM-dd');
+  } catch (error) {
+    if (process.env.NODE_ENV !== 'test') {
+      console.error(error);
+    }
+    return inputDate;
```

Comment:
question: is this not hiding a side effect? What date is failing?

Reply:
it will still log the error in console but prevent the whole app from crashing.

This is based on the screenshot you shared on slack with me. The error there is because of invalid date
![image](https://github.com/GenomicDataInfrastructure/gdi-userportal-frontend/assets/1210046/82359923-efcb-4429-a59a-86e4412c8c7d)